client/app/util/PerfDebug.js

```
@@ -67,7 +67,7 @@ export const timeFunctionPromise = (fn, onTimeElapsed, label = '') => (...args)
   const returnPromise = fn(...args);
```

The changes in this file were necessary because we were only timing HTTP requests for successful requests. Failures were being ignored.